plugins/callback/foreman.py

```diff
@@ -105,6 +112,9 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
 
         super(CallbackModule, self).set_options(task_keys=task_keys, var_options=var_options, direct=direct)
 
+        if self.get_option('disable'):
+            self._disable_plugin()
```

Comment:
[WARNING]: Skipping callback 'theforeman.foreman.foreman', unable to load due to: _disable_plugin() missing 1 required positional argument: 'msg'

Reply:
Oh, right, not really a graceful disable, will fix

Reply:
Done